pkg/controllers/monitoring-stack/controller.go

```diff
@@ -118,6 +119,10 @@ func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	for _, patcher := range patchers {
 		if err := r.reconcileObject(ctx, ms, patcher); err != nil {
+			if errors.IsAlreadyExists(err) || errors.IsConflict(err) {
```

Comment:
Can you pull this out of the if statement in order to avoid double nesting?
```
err := r.reconcileObject(ctx, ms, patcher)
if errors.IsAlreadyExists(err) || errors.IsConflict(err) {
...
}

if err != nil {
...
}

Reply:
yes, we definitely can and reads better